app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
⚠️ `||=` Shouldn't be used when there is possibility that statement will evaluate to `nil` or `false` because then we lose all the benefit of memoization. Instead it should be done like this:

```ruby
def current_user
  return @current_user if defined?(@current_user)
  @current_user = session[:user_id] && User.find_by(id: session[:user_id])
end
```